templates/getting_started/index.md

```diff
@@ -25,7 +25,8 @@ You're going to need more than a one-pager. And you're in luck:
 The simplest way to install Keras is to via `pip`:
 
 ```
-pip install keras
```

Comment:
oh, I think the pip install is correct here, we should update any "import keras" to "from tensorflow import keras"

Reply:
I think the user is complaining about `pip install keras` as that was not installing dependencies.

> pip install keras no longer installs the dependencies of keras (the install_requires is no longer correct in terms of setuptools metadata).